src/docbuilder/limits.rs

```diff
@@ -0,0 +1,97 @@
+use error::Result;
+use postgres::Connection;
+use std::collections::BTreeMap;
+use std::time::Duration;
+
+const DEFAULT_MEMORY_LIMIT: usize = 3 * 1024 * 1024 * 1024; // 3 GB
+const DEFAULT_TIMEOUT: Duration = Duration::from_secs(15 * 60); // 15 minutes
+const DEFAULT_NETWORKING: bool = false;
```

Comment:
Is it useful to have these separate from the default impl?

Reply:
Mostly because it's easier to see and change them when looking at the file. I don't care too much about them being there.